README.md

```diff
@@ -111,19 +111,19 @@ Rachael Stickland at 2024-01-05T13:22:09.774Z
 
 ℹ Found 2 Data Classes (2 tables) in this Data Asset
```

Comment:
It doesn't have to be on this line in particular but would be good to mention where the data assets are stored perhaps? for eg. the data assets are usually stored in data_raw and that's where you'd usually put them